packages/react-data-grid/src/ReactDataGrid.tsx

```diff
@@ -377,7 +392,7 @@ export default class ReactDataGrid extends React.Component<Props, State> {
     const fromRowData = rowGetter(action === UpdateActions.COPY_PASTE ? originRow! : fromRow);
     const fromRowId = fromRowData[rowKey];
     const toRowId = rowGetter(toRow)[rowKey];
-    onGridRowsUpdated({ cellKey, fromRow, toRow, fromRowId, toRowId, rowIds, updated, action, fromRowData });
+    onGridRowsUpdated({ cellKey, fromRow, toRow, fromRowId, toRowId, rowIds, updated: updated as never, action, fromRowData });
```

Comment:
This is bad.

To get the proper type on `onGridRowsUpdated` we need to add a generic on the Props object/RDG component so we can set a map of `{ [column key]: column value type }`, used like this:
```tsx
<ReactDataGrid<{ colKey: string; }> columns={[]} />
```
, but then we'd have to set a generic everywhere.

This'll do for now I think.

Reply:
Do you think we can just set it to `Record<string, unknown>` and let the parent component handle it. It makes sense because we do not know what editor is going to return and typing it makes grid more complicated